nixos/modules/services/web-apps/miniflux.nix

```
@@ -22,7 +22,7 @@ let
   pgbin = "${config.services.postgresql.package}/bin";
   preStart = pkgs.writeScript "miniflux-pre-start" ''
```

How about something like this:

```suggestion
  # The hstore extension is no longer needed as of v2.2.14
  # and would prevent Miniflux from starting.
  preStart = pkgs.writeScript "miniflux-pre-start" ''
```

---

Added comment.